src/app/components/Embeds/EmbedHtml/fixtures.js

```diff
@@ -0,0 +1,7 @@
+const quotePicEmbedHtml =
```

Comment:
Just to be super picky, could this file be `.ts`

Reply:
Accidentally did the commits in the AMP PR, all the comments will get addressed in there https://github.com/bbc/simorgh/pull/10965